# Workshop "Establishing a knowledge graph community in biomedical science"

!!! info "Workshop Information"
    :material-calendar-range: **Date:** 15-19.06.2026  
    :material-map-marker: **Place:** Heidelberg, Im Neuenheimer Feld 205

## Background

Many modern biomedical methods benefit from the availability of prior knowledge, for example about genes, proteins, or diseases. Knowledge graphs, i.e., representations of prior knowledge in machine-readable graph form, have become the quasi-standard for storing, manipulating, and sharing biomedical prior knowledge. To meet the needs of broad user communities in generating knowledge graphs, we have developed BioCypher, a modular framework for the creation of knowledge graphs based on ontologies, targeting single cell and spatial omics, microbiomics, metabolomics, and various multi-omics modelling and machine learning methods.

In this workshop, we will learn about knowledge representations, knowledge graphs, ontologies, and data structures, and put this knowledge to practical use with BioCypher. Further, we will learn how to leverage generative AI to upscale to more complex projects.

## Workshop Aim
    
Learn how to create knowledge graphs from your data and import them into a graph database for further studies.

## Research Topics

<div class="grid cards" markdown>

- :material-graph-outline: **Knowledge graphs**
- :material-dna: **Biomedical science**  
- :material-file-tree: **Ontologies**
- :material-vector-triangle: **Graph representations**
- :material-brain: **Knowledge extraction**
- :material-chat-processing-outline: **Agentic automation**

</div>

## Prerequisites

!!! warning "Requirements"
    Familiarity with ontologies and/or knowledge graphs and/or Python is helpful. You need to bring a laptop with a working Python installation and an IDE such as VSCode.

## Registration

[Register for the Workshop :material-arrow-right:](https://limesurvey.urz.uni-heidelberg.de/index.php/452584?lang=en){ .md-button .md-button--primary }
